packages/context-entries/lib/Keys.ts

```diff
@@ -48,6 +48,15 @@ export const KeysHttp = {
    * but until the response is fully consumed.
    */
   httpBodyTimeout: new ActionContextKey<boolean>('@comunica/bus-http:http-body-timeout'),
+  /**
+   * Number of retries to make on failed network calls. This only takes effect
+   * on errors thrown during the initial fetch() call and not while streaming the body.
```

Comment:
> and not while streaming the body.

I get why. But perhaps it should?
What do you think @jeswr?

Complexity of this PR would increase quite a bit with this though, so if we do this, we may want to do it in a separate PR.

Reply:
I agree we should tackle it at some point but I also think it should be a separate PR.


In order to do it properly we would have to add code to find the point in the retried stream at which the original stream broke and continue from there (since Comunica will have already started processing the first stream which broke; and we don't want to introduce broken results).

It would also be completely different error handling implementation wise because rather than throwing an error in the actor as we do here; we would instead need to be doing error handling on the stream.

Reply:
Yeah, it will not be trivial, that's for sure.
There's probably even a research paper in there :-)